RecoLocalCalo/HcalRecAlgos/interface/PulseShapeFunctor.h

```diff
@@ -0,0 +1,75 @@
+#ifndef PulseShapeFunctor_h
+#define PulseShapeFunctor_h 1
```

Comment:
Please use an include guard matching the full file name subsystem_package_fileName_h